docs/API.md

```diff
@@ -493,4 +494,89 @@ static flushGetRequests(): void
 
 **Returns**:
 
-`undefined`
\ No newline at end of file
+`undefined`
+
+
+<!-- ------------------------ HOOKS ------------------------ -->
+
+
+## `useAsyncStorage`
+
+Uses the new [hooks api](https://reactjs.org/docs/hooks-intro.html) to give you convenience functions to get, set, merge and delete a value from a location.
```

Comment:
What do you think about changing `from a location` to `from Async Storage, for given key` or something similar? Just to point that this custom hook is Async Storage key-specific.